buildSrc/src/main/groovy/io/micronaut/build/internal/ext/MicronautCoreExtension.java

```diff
@@ -28,4 +28,6 @@ public interface MicronautCoreExtension {
 
     void usesMicronautTestKotest();
 
+    void usesMicronautTestKotest5();
```

Comment:
can we just use kotest5 instaed of mantaining both?

Reply:
Gone

Reply:
why create a new method instead of keeping old one?

Reply:
Fixed in https://github.com/micronaut-projects/micronaut-core/pull/8074/commits/624810d8a859c728f53593a814bd9a3a5247d0e2